### DISCUSSION TITLE
Build Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /vercel/61b8ea43/next.config.js 😢

### DISCUSSION THREAD

**Discussion by amadrzyk** [vercel/next.js#15343] [Help]
Hi! I tried deploying my code to Vercel but randomly got this issue today:
<img width="1000" alt="Screen Shot 2020-07-20 at 12 44 22 PM" src="https://user-images.githubusercontent.com/20073334/87979343-c5babf00-ca86-11ea-9aaf-20b2c5f7d199.png">

Not entirely sure how to proceed with fixing this issue. Here is a link to the branch that is running into the issue, to reproduce the error: https://github.com/amadrzyk/url-metadata-parser/tree/change_method_to_post

Thank you!

**Comment by otitonium** [comment]
Did you finally solve this? I am also facing same issue right and has stalled my progress for the past few weeks.
